specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/InstancePools.json

```diff
@@ -376,18 +372,32 @@
           "x-ms-enum": {
             "name": "InstancePoolLicenseType",
             "modelAsString": true
-          },
-          "x-ms-mutability": [
-            "read",
-            "create"
-          ]
+          }
+        },
+        "dnsZone": {
+          "description": "The Dns Zone that the managed instance pool is in.",
+          "type": "string",
+          "readOnly": true
+        },
+        "maintenanceConfigurationId": {
```

Comment:
>maintenanceConfigurationId [](http://example.com/codeflow?start=9&length=26)

Should describe the kind/format of ID is this (resourceId, GUID, name, etc.). If this is an Azure resource ID, recommend adding the arm-id format + x-ms-arm-id-details. Ref: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details). If it's a guid, you can use format: uuid.